specs/~inheritance.yml

```diff
@@ -224,3 +224,92 @@ tests:
     partials:
       parent: "{{$foo}}default content{{/foo}}"
     expected: default content
+
+  - name: Standalone parent
+    desc: A parent's opening and closing tags need not be on separate lines in order to be standalone
```

Comment:
That is, this pair of tags behaves exactly like a single standalone partial tag.